Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
import matplotlib.pyplot as plt

# Add 1 mm diameter point
I0=np.linspace(10, 300, num=50)

# Scaling law
Rth=R_th_jc_ref*(I0/I0_ref)**(-1)

# plot
h, ax = plt.subplots(1,1,sharex=True)
ax.plot(I0, Rth, '--r',df['I0'],df['Rth_jc'],'.b', I0_ref, R_th_jc_ref, '.r')
ax.set_ylabel('Thermal resistance (junction to case) [Â°C/W]')
ax.set_xlabel('Rated current [A]')
ax.grid()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[3], line 8[0m
[1;32m      5[0m I0[38;5;241m=[39mnp[38;5;241m.[39mlinspace([38;5;241m10[39m, [38;5;241m300[39m, num[38;5;241m=[39m[38;5;241m50[39m)
[1;32m      7[0m [38;5;66;03m# Scaling law[39;00m
[0;32m----> 8[0m Rth[38;5;241m=[39m[43mR_th_jc_ref[49m[38;5;241m*[39m(I0[38;5;241m/[39mI0_ref)[38;5;241m*[39m[38;5;241m*[39m([38;5;241m-[39m[38;5;241m1[39m)
[1;32m     10[0m [38;5;66;03m# plot[39;00m
[1;32m     11[0m h, ax [38;5;241m=[39m plt[38;5;241m.[39msubplots([38;5;241m1[39m,[38;5;241m1[39m,sharex[38;5;241m=[39m[38;5;28;01mTrue[39;00m)

[0;31mNameError[0m: name 'R_th_jc_ref' is not defined

